pylxd/image.py

```diff
@@ -84,3 +84,33 @@ def export(self):
         """Export the image."""
         response = self.api.export.get()
         return response.content
+
+    def add_alias(self, name, description):
+        """Add an alias to the image."""
+        self.client.api.images.aliases.post(json={
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        # Update current aliases list
+        self.aliases.append({
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        return self
+
+    def delete_alias(self, name):
+        """Delete an alias from the image."""
+        self.client.api.images.aliases[name].delete()
+
+        # Update current aliases list
+        la = [a['name'] for a in self.aliases]
+        try:
+            del self.aliases[la.index(name)]
+        except ValueError:
+            pass
+
+        return self
```

Comment:
If you don't really need the feature, the issue is moot, and we could just delete the line.

It would seem awkward (and maybe non-pythonic) to do `image.delete_alias("a").delete_alias("b")`. You aren't making any better code than something like:

    image.delete_alias('a')
    image.delete_alias('b')

...or better yet...

    for alias in ['a', 'b']:
        image.delete_alias(alias)